src/QueryBuilder.php

```diff
@@ -445,6 +445,24 @@ public function whereType($documentType, $boolean = 'and')
         return $this;
     }
 
+    /**
+     * @param string $name
+     * @param mixed  $id
+     * @param string $boolean
+     * @return QueryBuilder
+     */
+    public function whereParentId(string $name, $id, string $boolean = 'and'): self
+    {
+        $this->wheres[] = [
+            'type' => 'ParentId',
+            'name' => $name,
```

Comment:
is `name` the name/type/index of the parent? Worth making that clearer somewhere either with a comment or a different variable name.